components/director/internal/tenantfetchersvc/resync/event_page.go

```diff
@@ -125,7 +125,15 @@ func (ep EventsPage) eventDataToMovedSubaccount(ctx context.Context, eventData [
 		log.C(ctx).Warnf("Missig or invalid format of field: %s for tenant with ID: %s", ep.FieldMapping.SubdomainField, id)
 	}
 
-	subaccountInput, err := constructSubaccountTenant(ctx, jsonPayload, nameResult.String(), subdomain.String(), id, ep)
+	licenseType := gjson.Get(jsonPayload, ep.FieldMapping.LicenseTypeField)
+	var licenseTypeValue *string
+	if licenseType.Exists() && licenseType.Type == gjson.String {
+		licenseTypeValue = &licenseType.Str
+	} else {
+		log.C(ctx).Warnf("Missing or invalid format of licenseType field: %s for tenant with ID: %s", ep.FieldMapping.LicenseTypeField, id)
+	}
+
+	subaccountInput, err := constructSubaccountTenant(ctx, jsonPayload, nameResult.String(), subdomain.String(), id, licenseTypeValue, ep)
```

Comment:
Do the subaccounts have license?

Reply:
Not really, they are with the license of its global account.